docs/source/tagset/elemento-contrib-group.rst

```diff
@@ -3,19 +3,27 @@
 <contrib-group>
 ---------------
 
-Aparece em
+Aparece em:
   :ref:`elemento-article-meta`
  
-Ocorre
+Ocorre:
   Uma vez
 
 
-Representa o grupo dos que contribuiram para a elaboração do artigo. 
-Os tipos de contribuintes mais frequentes são de autores pessoais, 
-instituições e grupos de pesquisa. A tag pode ou não envolver a 
-informação de afiliação, sendo obrigatória na identificação do contribuidor 
-do tipo autores (author) sejam institucionais ou não. Os principais 
-elementos de ``<contrib-group>`` são: :ref:`elemento-contrib`, 
-:ref:`elemento-xref`, :ref:`elemento-collab`, :ref:`elemento-aff` e 
-:ref:`elemento-role`.
+Encerra o grupo de elementos relativos à contribuição na elaboração
+do artigo. Os contribuintes mais frequentes são os autores pessoais, 
+instituições e grupos de pesquisa.
 
```

Comment:
O texto fica mais claro quando lido no seu conjunto, uma vez que as informações foram distribuídas em parágrafos menores. Veja abaixo:

Encerra o grupo de elementos relativos à contribuição na elaboração
do artigo. Os contribuintes mais frequentes são os autores pessoais,
instituições e grupos de pesquisa.

O elemento pode ou não informar a afiliação, sendo obrigatória quando
o atributo tipo de contribuição (@contrib-type) for ``author``, sejam
estes autores institucionais ou não.

Os principais elementos de ``<contrib-group>`` são:

* :ref:`elemento-contrib`;
* :ref:`elemento-xref`;
* :ref:`elemento-collab`;
* :ref:`elemento-aff`; e
* :ref:`elemento-role`.